src/data/faq_de.json

```diff
@@ -325,6 +325,14 @@
                         "anchor": "minimum_requirements",
                         "active": true,
                         "textblock": [
+                            "<b>AKTUELL 8. Februar 2021</b><br/>Seit dem 5. Februar 2021 werden nur die Corona-Warn-App (CWA) Versionen 1.5 und höher unterstützt. Aus diesem Grund erhalten Nutzer mit älteren Versionen einen Hinweis, Ihre App entsprechend zu aktualisieren. Dies gilt sowohl für iOS als auch Android. Parallel gelten jedoch bestimmte Mindestvoraussetzungen an das Betriebssystem: Android mindestens 6.0 und iOS mindestens 13.7.",
+                            "iOS Nutzer, die bisher bspw. iOS 13.5 genutzt haben, müssen Ihr iOS Betriebsystem nun zwingend auf mindestens iOS 13.7 aktualisieren, um die CWA weiterhin zu nutzen. Nur in diesem Fall kann eine technisch korrekte Funktion der App sichergestellt werden.",       
+                            "<ul><li>Bitte führen Sie zunächst ein Update Ihrer iOS-Version durch. Auch ältere Smartphones, wie bspw. das iPhone SE, 6S oder 6S Plus unterstützen das neueste iOS Betriebssystem. Sie können Ihr iPhone somit mindestens auf iOS 13.7 aktualisieren. Aus Sicherheitsgründen sollten Sie ihr iPhone immer aktuell halten, da Apple auch regelmäßig kleinere Sicherheitslücken mit den Updates behebt. Sie können dies auf Ihrem iPhone unter Einstellungen – Allgemein – Softwareupdate durchführen.</li><li>Im Anschluss aktualisieren Sie bitte die Corona-Warn-App. Die Corona-Warn-App muss zudem mindestens in Version 1.5 installiert sein. Inzwischen ist bereits Version 1.11 verfügbar. Wir bitte Sie daher, Ihre Corona-Warn-App zu aktualisieren.</li> </ul>",
+                            "Ab der CWA Version 1.12 werden zusätzlich auch die ältere iPhone Modelle iPhone 5s, iPhone 6, iPhone 6plus unterstützt. Diese Geräte müssen auf die iOS Version 12.5 bzw. 12.5.1 aktualisiert werden",

```

Comment:
iPhone 5s, iPhone 6, iPhone 6plus => iPhone 5s, iPhone 6 bzw. iPhone 6plus
Diese Geräte müssen auf die iOS Version 12.5 bzw. 12.5.1 aktualisiert werden (Full stop missing at end of sentence)

Reply:
ok